starter-code/scripts/articleView.js

```diff
@@ -64,7 +58,11 @@ articleView.handleMainNav = function() {
   //       single .tab-content section that is associated with the clicked .tab element.
   //       So: You need to dynamically build a selector string with the correct ID, based on the
   //       data available to you on the .tab element that was clicked.
-
+  $('.main-nav ul').on('click', 'li', function(){
```

Comment:
If you read the `TODO` comment again, you will notice that it asks you to use `.tab` element.
i.e `$('.main-nav').on('click', '.tab', function(){`

If now you click on `Github` and watch what happens on your page.
In the html file, we have
````
<li class="tab" data-content="articles"><a href="#" class="icon-home"> Home</a></li>
<li class="tab" data-content="about"><a href="#" class="icon-address-book"> About</a></li>
<li><a href="http://twitter.com/brookr" target="_blank" class="icon-twitter3"> Twitter</a></li>
<li><a href="http://github.com/brookr" target="_blank" class="icon-github"> GitHub</a></li>
````
both Twitter and GitHub don't have `data-content`.
And that's why we only want to handle event when the element class is `tab`.